src/views/passwordReset/RequestPasswordReset.jsx

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable react/jsx-no-bind */
+import React, { useState } from 'react';
+import {
+  Container,
+  Row,
+  Col,
+  Form,
+  Alert
+} from 'react-bootstrap';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Button from '../../components/button/Button';
+import { requestPasswordRequest } from '../../redux/actions/passwordReset';
+
+const RequestPasswordReset = (props) => {
+  const [state, setState] = useState({
+    email: null,
+    error: null,
+    message: null
+  });
+
+  const updateInput = e => setState({
+    ...state,
+    email: e.target.value,
+    error: null,
+    message: null
+  });
+
+  const completeRequest = async (e) => {
+    e.preventDefault();
```

Comment:
@youngestdj what is `e`? You should make your params readable. Use `event` instead....and please pass this to the team

Reply:
I have fixed this @9jaswag